.pipelines/templates/c-api-ios-cpu.yml

```diff
@@ -0,0 +1,151 @@
+# packaging pipeline for iOS CocoaPods package
```

Comment:
can we maybe call it build-ios-cocoapods-package.yml or something like that? would be good to consider all the yml file names and try and give them a meaningful name for what they do.

Reply:
Renamed to build-package-for-xxx.yml